.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Chromatic'
+on: push
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "12"      
```

Comment:
Should it be aligned with `"devEngines"` from our `package.json`?

Reply:
No, forget what package.json=>devEngines is defined it is just decoration. The only what counts is https://github.com/input-output-hk/daedalus/blob/66f9ea0cd87e44fc58801d5dc9d0542f4e24cc42/default.nix#L58 🤷‍